package.json

```diff
@@ -30,24 +30,28 @@
     "minimist": "1.2.6",
     "ndb": "1.1.5",
     "nodemon": "2.0.19",
-    "prettier": "2.6.2"
+    "prettier": "2.6.2",
+    "supertest": "^6.2.4"
   },
   "dependencies": {
     "bcrypt": "5.0.1",
+    "chai": "^4.3.6",
     "chalk": "4.1.2",
     "cors": "2.8.5",
     "dotenv": "16.0.1",
     "express": "4.18.1",
     "joi": "17.6.0",
     "jsonwebtoken": "8.5.1",
     "lodash": "4.17.21",
+    "mocha": "^10.0.0",
```

Comment:
@beblicarl this should be a dev dependency. same as the others

Reply:
yes please, i have made the change